src/admin/admin.service.ts

```
@@ -2926,4 +2927,166 @@ export class AdminService {
+
+    // Unique stories accessed
+    const storyLogs = await this.prisma.activityLog.findMany({
+      where: { action: 'GUEST_STORY_ACCESSED', isDeleted: false },
+      select: { details: true },
+    });
+    const uniqueStoryIds = new Set<string>();
+    for (const log of storyLogs) {
+      try {
+        const parsed = JSON.parse(log.details ?? '{}');
+        if (parsed.storyId) uniqueStoryIds.add(parsed.storyId);
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)